src/components/Chip/Chip.tsx

```diff
@@ -34,48 +36,47 @@ class Chip extends React.PureComponent<Props, any> {
       theme,
     } = this.props;
 
-    let className;
+    const className = classNames(
+      theme.Chip,
+      transparent && theme.transparent
+    );
 
-    if (this.props.markedForDelete) {
-      className = classNames(
-        theme.Chip,
-        theme.markedForDelete,
-        transparent && theme.transparent
-      );
-    } else {
-      className = classNames(
-          theme.Chip,
-          transparent && theme.transparent
-      );
-    }
+    const deleteInstruct = this.props.removable ? 'Press delete to remove this chip' : '';
```

Comment:
@weegrom Do you have any opinion on the wording of this text. We have this hidden in text below, probabaly for the readers.  Is team India going to test this kind of functionality? @krunalm

Reply:
@harsimranb It looks like this text was borrowed from Angular, so I'm okay with the wording.

@harsimranb and @krunalm  Yes, we do want the team testing accessibility.